examples/quickstarts/helidon-quickstart-graalvm/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM helidon/java-native:0.1.13 as build-native-image
```

Comment:
Where is the source for `helidon-java-native:0.1.13` ? Did tomas put it in the workspace yet ?

Reply:
Probably yes because my default docker installation got this image without any problems

Reply:
Dockerbuild does not resolve base images from source. Tomas did push this image to Dockerhub. The question is to prompt him to add the source to the workspace.

Reply:
He's on vacation so this won't be known this week.

Reply:
ok.